### DISCUSSION TITLE
Directive that provides attribute hlmCardTitle is out of scope of the current template

### DISCUSSION THREAD

**Discussion by CaetanoJS** [spartan-ng/spartan#280] [Q&A]
Hello everyone, i've been using spartan a while for a personal project and i receive this message for almost all spartan directives that i add in my html. It still works but it is a pain in the ass to see this message in my IDE all the time (i'm using WebStorm latest version).

The fix that my IDE recommends to import the directive, but it is already imported, so the IDE duplicates the import but the message stills there:

```typescript
@NgModule({
  declarations: [
    appComponent
  ],
  imports: [
    HlmCardTitleDirective,
    HlmCardTitleDirective,
  ],
})
```

my spartan files are added in the following folder path (i'm using npm):

```json
  "@spartan-ng/ui-card-helm": [
    ".\\spartan/ui-card-helm/src/index.ts"
  ],
```

i already tried to add it inside src/app folder, but with no success.

anyone has saw something like this while working with spartan and intelij IDE?

**Comment by goetzrobin** [comment]
Hey there! Interesting. Can you share a screenshot of your IDE with the error. I'm using webstorm also, but haven't noticed that
